src/shared/layouts/ProtectedNavBarLayout/ProtectedNavBarLayout.tsx

```
@@ -0,0 +1,19 @@
+import React, { PropsWithChildren, ReactElement } from 'react'
+
+import { NavBarLayout, RootLayout } from '@/shared/layouts'
+import { NextPage } from 'next'
+
+import { ProtectedWrapper } from './ProtectedWrapper'
+
+export const ProtectedNavbarLayout: NextPage<PropsWithChildren<{}>> = ({ children }) => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)